.release-please-manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "cockroachdb": "1.0.0",
+  "elasticsearch": "1.0.0",
+  "es-topic-indexer": "1.0.0",
+  "redis": "1.0.0",
+  "redis-cluster": "1.0.0"
```

Comment:
Since we are forced to use semver with release-please, I think we can start from 1.0.0.
Before that, we used versions like `"image version tag"-1` for cockroach manifest.

Reply:
> we used versions like "image version tag"-1 for cockroach manifest.

Ah right, now I remember, trying to match this format was the painful bit when trying to use `release-please` in the cockroachdb manifests. Do we still need to surface the version of the underlying tech somewhere? E.g. if someone is running `vX` of cockroachDB how would they know which version of the manifests in this repo are compatible with that? Maybe that information just lives in the changelog?

Reply:
>  Maybe that information just lives in the changelog?

Yeah, I agree with that. Doesn't seem like anyone would like to maintain new versions of manifests for the old version of the image - and even if this situation will happen, I don't think we should base our release system on that. 

I'm kind of uneasy with versioning all the existing software as 1.0.0, though. But I don't know which version else could we use.

Reply:
Indeed, I just found the slack convo https://utilitywarehouse.slack.com/archives/C03LB7ZN07J/p1718869775335839?thread_ts=1718781427.663029&cid=C03LB7ZN07J

TLDR we were thinking that something like `redis-v7.2.0-base-v1.0.0` would work.

Reply:
> TLDR we were thinking that something like redis-v7.2.0-base-v1.0.0 would work.

I don't think that format works well with `release-please` https://github.com/utilitywarehouse/kafka-manifests/pull/41. If we don't plan on supporting more than one version of each tech (IIRC from recent discussions, e.g. we don't want manifests for both Redis v7 and Redis v6) do we need/want this tag format (basically: do we want to go through the extra effort to try and make that tag format work with `release-please`?)

Reply:
Here's a summary of our sync discussion:
- we're going with offering a "full" product package for now with keeping the tags as component-x.y.z without exposing the component version in the tag. 
- The release changelog will specify which version of the component is supported in that release
- we're gonna present this option at TAG and see what people think, and we can adjust afterwards

Reply:
:) didn't notice that